src/Model.js

```diff
@@ -9,9 +15,10 @@ export default class Model {
     }
 
     this.configure && this.configure()
-    this[exec_middlewares]('before', 'create')
-    // TODO: create model
-    this[exec_middlewares]('after', 'create')
+    this[execute_middleware]('before', 'create')
+    this[collection] = this.constructor.collection
+    this.id = this[collection].id()
```

Comment:
I think its better to focus on mongo 100% right now. However I think id should be present in every document as ActiveRecord do even though its not an Mongo Document.